modules/local/input_assure/main.nf

```diff
@@ -0,0 +1,32 @@
+process INPUT_ASSURE {
+    tag "Assures Inputs are Consistent"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.8.3' :
+        'biocontainers/python:3.8.3' }"
+
+    input:
+    tuple val(meta), path(mlst)
+
+    output:
+    tuple val(meta), path("${meta.id}.mlst.json.gz"),               emit: result
+    tuple val(meta), path("*_error_report.csv"), optional: true,    emit: error_report
+    path("versions.yml"),                                           emit: versions
+
+    script:
+
+    """
+    input_assure.py \\
+        --input ${mlst} \\
+        --sample_id ${meta.id} \\
+        --address ${meta.address} \\
```

Comment:
Note that `meta.address` is always null for gasclustering (it sometimes is null and sometimes is not for gasnomenclature). Because of the logic of `input_assure.py` though, this still works. So, I don't think it needs to be changed.